examples/tutorials/first-figure.py

```diff
@@ -34,11 +35,13 @@
 fig = pygmt.Figure()
 
 ########################################################################################
-# Add elements to the figure using its methods. For example, let's start a map with an
-# automatic frame and ticks around a given longitude and latitude bound, set the
-# projection to Mercator (``M``), and the map width to 8 inches:
+# Add elements to the figure using its methods. For example, let's use
+# :meth:`pygmt.Figure.basemap` to start a map. We'll use the ``region`` parameter
+# provide the longitude and latitude bounds, the ``projection`` parameter to set
+# the projection to Mercator (**M**) and the map width to 8 cm, and the ``frame``
+# parameter to generate a frame with automatic tick and annotation spacings.
 
-fig.basemap(region=[-90, -70, 0, 20], projection="M8i", frame=True)
+fig.basemap(region=[-90, -70, 0, 20], projection="M8c", frame=True)
```

Comment:
The default settings (e.g., annotation font size, frame width) look nicer for 15-cm-wide maps.
```suggestion
fig.basemap(region=[-90, -70, 0, 20], projection="M15c", frame=True)
```